lib/src/animate.dart

```diff
@@ -346,6 +353,7 @@ extension AnimateWidgetExtensions on Widget {
         controller: controller,
         adapter: adapter,
         target: target,
+        useRepaintBoundary: useRepaintBoundary ?? false,
```

Comment:
I try really hard to not duplicate defaults in extension methods, since it creates fragility when changing them (easy to forget to update in all locations). Take a look at any of the extension methods on various effects for examples.